backend/app/services/memory/client.py

```
@@ -0,0 +1,375 @@
+
+        except asyncio.TimeoutError:
+            logger.warning(
+                "Memory service health check timed out after %.1fs", self.timeout
+            )
+            return False
+        except aiohttp.ClientError as e:
+            logger.warning(
+                "Memory service health check failed (connection error): %s", e
+            )
+            return False
+        except Exception as e:
```

Ruff (0.14.13)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)